### DISCUSSION TITLE
avoid closing the DropdownMenu if the user clicks outside the DropdownMenu

### DISCUSSION THREAD

**Discussion by pavelgromyko** [radix-ui/primitives#2470] [Help] [answered]
At the moment I'm using DropdownMenu. One nuance arose in the work.
I open a DropdownMenu and if I click with the cursor outside the boundaries of the DropdownMenu, the DropdownMenu closes. I have a special button that allows the user to close a window.
Tell me how to avoid closing the DropdownMenu if the user clicks outside the DropdownMenu?

**Comment by joaom00** [comment, marked as answer]
You can use `onInteractOutside={event => event.preventDefault()}` in `DropdownMenu.Content`

https://www.radix-ui.com/primitives/docs/components/dropdown-menu#content

**Reply by pavelgromyko** [reply]
Thanks, I should be more careful!!!
